resources/views/service/index_registration.blade.php

```diff
@@ -40,7 +40,7 @@
                     @foreach ($registrations as $registration)
                         <tr>
                             <td>{{ $registration->family->carers->first()->name }}</td>
-                            <td>{{ $registration->cc_reference }}</td>
+                            <td>{{ $registration->family->rvid }}</td>
```

Comment:
From prototyping, voucher entitlement is the second col in the div and then RV ID. As I've already changed this, could just remove the changes to this file?